test/object-store/sync/app.cpp

```diff
@@ -4273,13 +4419,17 @@ TEST_CASE("app: base_url", "[sync][app][base_url]") {
             session->resume();
             state.wait_for(TestState::session_started);
 
-            CHECK(redir_transport->location_requested);
-            CHECK(app->get_base_url() == init_url);
-            CHECK(app->get_host_url() == redir_url);
-            CHECK(app->get_ws_host_url() == redir_wsurl);
-            CHECK(app->sync_manager()->sync_route());
-            CHECK(app->sync_manager()->sync_route()->find(redir_wsurl) != std::string::npos);
+            //            CHECK(redir_transport->location_requested);
```

Comment:
?

Reply:
I forgot to add these back in for a final sanity check at the end of the test